examples/auth0/README.md

```diff
@@ -0,0 +1,65 @@
+# Next.js and Auth0 Example
+
+This example shows how you can use `@auth0/nextjs-auth` to easily add authentication support to your Next.js application.
```

Comment:
Seems like the `yarn create next-app` part is missing in this readme.

Reply:
Added